feat(docs): dev section in README.md (#76)
WFCD/warframe-drop-data: README.md

}
```

## For Developers

### Dependencies
- [Node.js >= 14](https://nodejs.org/en/)

### Usage

> You can run the build script with `npm run start`.

Reviewer: this won't copy the built data, that's what `npm build` is for... also `run` is extraneous on `start`, since it's built-in

```suggestion
You can run the build script with `npm run build`.
```

> You can run the build script with `npm run start`.

Reviewer: this won't copy the built data, that's what `npm build` is for... also `run` is extraneous on `start`, since it's built-in

```suggestion
You can run the build script with `npm run build`.

```

> Once built you serve the app Web Ui with `npm run serve:dev` or `npm run serve`

Reviewer: ```suggestion
Once built, serve the Web UI with `npm run serve:dev` or `npm run serve`.
```

> **The npm run serve needs a $PORT in your terminal env*

Reviewer: ```suggestion
*The `npm run serve` script requires the `PORT` environment variable to be defined. `npm run serve:dev` provides this through nodemon.*
```

Reviewer: I'm also a bit concerned because `npm run serve` is for deployment, not development

Author: Yeah the dev script solves this issue, it just doesn't in windows which would be another issure apart.

Reviewer: Runs fine I'm my windows environment
